website/docs/components/separator/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,30 @@
+
+## Placement
+
+Use separators to distinguish between sections on a page visually. Use them sparingly and purposefully to avoid cluttering the UI and distracting users from the main content.
+
+!!! Do
+
+Use to create a clear break between paragraphs, blocks, or sections.
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-do.png)
+
+!!!
+
+!!! Dont
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-dont.png)
```

Comment:
As "consumer" I am not sure how to interpret this "don't" image: to me it seems very similar to the "do" one, I am not sure what I should **not** do.

Reply:
@didoo This might've been before the update to the 'Do & Don't' section, but what do you think now with the added description? [Separator Website Link](https://hds-website-git-alex-ju-add-separator-docs-hashicorp.vercel.app/components/separator)

Reply:
@heatherlarsen Yes, I have some from the light audit I can fix for the examples. Let me get those in.